src/module/actor/character/sheet.ts

```diff
@@ -1159,6 +1159,13 @@ class CharacterSheetPF2e<TActor extends CharacterPF2e> extends CreatureSheetPF2e
         return handlers;
     }
 
+    protected override _onChangeTab(event: MouseEvent, tabs: Tabs, active: string): void {
+        super._onChangeTab(event, tabs, active);
+        for (const tab of htmlQueryAll(tabs["_nav"], "[role=tab]")) {
```

Comment:
```suggestion
        for (const tab of htmlQueryAll(tabs._nav, "[role=tab]")) {
```

Reply:
It won't let me :( I think it's because it's marked as `protected` in the type definition.

Reply:
Yeah, relaxing the access modifier for the `_nav` field from `protected` to `public` will let me access it directly. Want me to change the type definitions?

Reply:
It doesn't look like it's actually marked as protected in foundry.js

Reply:
Nope, the underscore is the only hint that it might possibly be considered protected or private. I've changed our type definition to reflect this.